src/server/services/data-transform.service.js

```diff
@@ -78,6 +78,7 @@ const transformAnswersForSummary = (cumulativeFullAnswers, addressLookups) => {
     if (data.operator_address_selected) {
       if (data.operator_address_line_1) {
         delete data.operator_address_selected;
+        data.operator_first_line = data.operator_address_line_1;
```

Comment:
This is the part that executes when the address is selected from Postcoder and has already been populated - I'm not sure when this occurs to be honest (if ever). You'll notice below, in the else, that data.operator_first_line is already set in the intital population from Postcoder so this additional line is not required. It's only the manual address entry that doesn't work ( the else of if (data.operator_address_selected) as you have amended below)